typeDefinition/constants/DomElementProps.d.ts

```diff
@@ -0,0 +1,75 @@
+export type DomElementProps = {
```

Comment:
I would not name it `DomElementProps`, there not only DOM elements involved, but also xbox/tvOS/Android. Maybe just remove `Dom` part